src/components/molecule_word_card/index.shared.tsx

```diff
@@ -20,6 +21,7 @@ interface Props {
 const URL_PATH = `/` + PageConst.Share + `?` + PageQueryConst.wordID + `=`
 
 const WordCardShared: FC<Props> = ({ wordId }) => {
+  const { width } = useWindowSize()
```

Comment:
dont forget to delete

Reply:
Handled in [65825d0](https://github.com/ajktown/wordnote/pull/205/commits/65825d09cd61c7debbdc17e9cffc23c128200fb6)